media-libs/amf-headers/amf-headers-1.4.34.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/GPUOpen-LibrariesAndSDKs/AMF"
+else
+	SRC_URI="https://github.com/GPUOpen-LibrariesAndSDKs/AMF/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="amd64 ~arm64"
```

Comment:
You're pushing straight to stable, **ekeyword ~all your.ebuild**.

Or is this another pre-compiled binary?

Reply:
I'll mark it unstable and not drop the previous version which is stable, did not notice

Reply:
So, from what I've just discovered: both `amf-headers` and `amdgpu-pro-amf` have been stabilized in the version `1.4.33` and I just dropped both. Should I keep them ?